### ISSUE TITLE
Swagger UI - option to change the request url host

### ISSUE THREAD

**Issue by dkirrane** [swagger-api/swagger-ui#1343]
It would be nice if you could change the host from Swagger UI in order to point at a different server when using the `Try it out!` button.

**Comment by sharmaak** [comment]
Much required.

**Comment by fehguy** [comment]
see https://github.com/swagger-api/swagger-js/issues/532

**Comment by webron** [comment]
Fixed in 2.1.1.

**Comment by rohanjoseph** [comment]
Hi i am not able to change the request URL in the swagger ui .it's always pointing to the localhost. 127.0.0.1:8080. I have hosted it in the amazon. so finally it should fetch the public ip but it is fetching the localhost ip which makes it difficult for us to work.
![images](https://cloud.githubusercontent.com/assets/5681794/11031118/3d9c63b6-86f8-11e5-8fe4-01e12f277939.png)

**Comment by andremilk** [comment]
I would like this feature very much too.

**Comment by webron** [comment]
@andremilk - feature already exists - https://github.com/swagger-api/swagger-js/issues/532.

**Comment by DennisYufei** [comment]
sorry，i'm new to swagger ui，and i want change the base url so that i can connect not only one host url ,but i don't know change files, please tell me how many files i should change, many thanks

**Comment by MaoJianwei** [comment]
```
It would be nice if you could change the host from Swagger UI in order to point at a different server when using the Try it out! button.
```

This problem is not resolved in swagger-editor:v3.17.0
